dataland-frontend/tests/e2e/specs/prepopulation/AwaitPrepopulation.ts

```diff
@@ -0,0 +1,30 @@
+import { CompanyInformation, EuTaxonomyData } from "../../../../build/clients/backend/api";
+import { retrieveCompanyIdsList } from "../../utils/ApiUtils";
+
+describe("I want to ensure that the prepopulation has finished before executing any further tests", () => {
+  let companiesWithData: Array<{ companyInformation: CompanyInformation; euTaxonomyData: EuTaxonomyData }>;
+  before(function () {
+    cy.fixture("CompanyInformationWithEuTaxonomyData").then(function (companies) {
+      companiesWithData = companies;
+    });
+  });
+
+  it(
+    "Should wait until prepopulation has finished",
+    {
+      retries: {
+        runMode: 250,
+        openMode: 250,
+      },
+    },
+    () => {
+      cy.wait(5000)
+        .then(() => retrieveCompanyIdsList())
+        .then((ids) => {
+          if (ids.length < companiesWithData.length) {
+            throw Error(`Only found ${ids.length} companies (Expecting ${companiesWithData.length})`);
+          }
+        });
+    }
+  );
+});
```

Comment:
Ist das der Fix für die Timeout-Probleme?

Reply:
Aktuell starten wir 4 parallele Test-Images. Allerdings muss in der CD nur einer dieser Parallelen runner die Prepopulation machen. Dieser Testcase wird als erstes von den 3 übrig gebliebenen Runnern ausgeführt, die Damit auf das Fertigwerden der Prepopulation warten, bevor die eigentlichen Tests ausgeführt werden.

Reply:
Ok